iocore/cache/Cache.cc

```diff
@@ -41,6 +41,8 @@
 
 constexpr ts::VersionNumber CACHE_DB_VERSION(CACHE_DB_MAJOR_VERSION, CACHE_DB_MINOR_VERSION);
 
+constexpr ts::VersionNumber CACHE_DB_VERSION_4847(24, 1);
```

Comment:
I think this is a poor choice of name - I have no idea what "4847" means. Perhaps something like "CACHE_DB_MIN_VERSION" or "CACHE_DB_OLDEST_VERSION".